### DISCUSSION TITLE
Custom orders through 3rd-party strategy

### DISCUSSION THREAD

**Discussion by ItsFlorkast** [Drakkar-Software/OctoBot#2538] [Q&A]
Hello everyone !

I was wondering if I could execute orders from my own strategy that are in another python script, it is possible? I've searched in the documentations but I haven't found anything related to my queestion.

**Comment by Herklos** [comment]
Hi @ItsFlorkast, did you have a look at [customize-your-octobot](https://www.octobot.cloud/en/guides/octobot-tentacles-development/customize-your-octobot)?

**Comment by ItsFlorkast** [comment]
Hello, 

Thanks for replying!

the documentation is really not clear to me & there are nothing related to making orders with specific stoplosses and so on. What I've found is the OctoBot-script and I think its related to my request. However, it is still not updated to 1.0.8 Octobot so i'm not being able to install it as I'm getting this message 

```
The conflict is caused by:
    octobot-script 0.0.16 depends on OctoBot==1.0.6
    octobot-script 0.0.15 depends on OctoBot==1.0.4
    octobot-script 0.0.14 depends on OctoBot==1.0.3
    octobot-script 0.0.13 depends on OctoBot==1.0.0

```

Any idea on how to bypass the conflict? Also, the orders created by Octobot-script will be linked to the OctoBot or works seperately?

**Reply by Herklos** [reply]
Hello,

Yes you can bypass the conflict, it will be fixed in the next octobot script release. 

No OctoBot script is dedicated to strategy creation and test. Once you have created and tested your strategy you can use it in OctoBot with paper or your real money.
